hr_expense_line_supplier/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+HR Expense Line Supplier
+========================
+
+This module adds a supplier field on the expense line.
+
+A second module will automatically install after this one, setting the field
+as 'required'. We need to separate this in 2 modules because we need the demo
+expense lines to have a supplier before setting the field required. This is
+done in order to avoid warnings on installation with demo data activated.
```

Comment:
@JulieSFL don't create another module to add demo data. This is overkill for the use of this module. It's not a big deal to have the warning because of the demo data.